container/service_options_test.go

```diff
@@ -1,125 +0,0 @@
-package container
```

Comment:
Any reason to delete the tests of this files?

Reply:
Yes. It is tested with the docker build test. No need to test it twice. In fact I wanted to get rid of those options and just pass SwarmSpec, but I thought you want to keep it simplified.